lib/vector-tap/src/controller.rs

```diff
@@ -451,13 +451,15 @@ async fn tap_handler(
     debug!(message = "Stopped tap.", outputs_patterns = ?patterns.for_outputs, inputs_patterns = ?patterns.for_inputs);
 }
 
-#[cfg(all(
+// FIXME: These these should be merged into `src/api/tests.rs` where _most_ of them were moved.
```

Comment:
Can you add a bit more context on why change is needed and when we can fix it?

Reply:
All of those feature flags are not present in `lib/vector-tap`, and the compiler now fails to build code with undeclared feature flags. With those flags removed, the code effectively is never compiled. The fix would be part of resolving #21487.

Reply:
Thanks!